### PR TITLE
Added logs

### PR DESCRIPTION
**Pull request by apereverzin**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/DTSBPS-879

### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: src/main/java/uk/gov/hmcts/reform/sendletter/services/LetterService.java [hunk 1]
```diff
@@ -110,7 +110,7 @@ public UUID save(ILetterRequest letter, String serviceName, String isAsync) {
 
     private UUID saveNewLetter(ILetterRequest letter, String messageId, String serviceName, String isAsync) {
         UUID id = UUID.randomUUID();
-        log.info("letterId {}", id);
+        log.info("letterId {}, thread id {}", id, Thread.currentThread().getId());
```
Comment (mdayican) [inline src/main/java/uk/gov/hmcts/reform/sendletter/services/LetterService.java:113]: ThreadName should be in the logs. why do you need Id?

**Review by mdayican — COMMENTED** [review]
it already logs the thread names.
Screenshot 2022-03-08 at 09 11 00

**Comment by apereverzin** [comment]
Indeed

**Comment by apereverzin** [comment]
Not necessary

**Pull request closed without merging**